turbinia/workers/volatility.py

```diff
@@ -46,7 +46,7 @@ def run(self, evidence, result):
 
     # TODO: Add in config options for Turbinia
     cmd = 'python2 /bin/vol -f {0:s} --profile={1:s} {2:s} \
```

Comment:
Pls add `--plugins=/evidence/plugins` in here. It needs to be added as the first argument after `/bin/vol`.